zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java

```diff
@@ -413,6 +413,7 @@ private Note loadNoteFromRepo(String id, AuthenticationInfo subject) {
       if (p.getDateFinished() != null && lastUpdatedDate.before(p.getDateFinished())) {
         lastUpdatedDate = p.getDateFinished();
       }
+      p.clearRuntimeInfo();
```

Comment:
Why we need to clear RuntimeInfo when we load note if the runtimeInfo is not stored ?